ui-participant/src/hub/task/taskUtils.ts

```diff
@@ -0,0 +1,69 @@
+import { Enrollee, HubResponse, ParticipantTask } from '@juniper/ui-core'
+
+/** returns the next actionable task for the enrollee, or undefined if there is no remaining task */
+export function getNextTask(enrollee: Enrollee, sortedTasks: ParticipantTask[]) {
+  const nextTask = sortedTasks.find(task => isTaskAccessible(task, enrollee) && isTaskActive(task))
+  return nextTask
+}
+
+export const TASK_TYPE_ORDER = ['CONSENT', 'SURVEY']
+export const TASK_STATUS_ORDER = ['IN_PROGRESS', 'NEW', 'COMPLETE']
+
+/** Sorts tasks based on their types, then based on status, and then based on their internal ordering */
+export function taskComparator(taskA: ParticipantTask, taskB: ParticipantTask) {
+  const typeOrder = TASK_TYPE_ORDER.indexOf(taskA.taskType) - TASK_TYPE_ORDER.indexOf(taskB.taskType)
+  if (typeOrder != 0) {
+    return typeOrder
+  }
+  const statusOrder = TASK_STATUS_ORDER.indexOf(taskA.status) - TASK_STATUS_ORDER.indexOf(taskB.status)
+  if (statusOrder != 0) {
+    return statusOrder
+  }
+  return taskA.taskOrder - taskB.taskOrder
+}
```

Comment:
I'd love just a quick test for both the task comparator and `getNextConsentTask`